articles/consuming-unsplash-api-using-node.js-graphql-api/index.md

```diff
@@ -0,0 +1,436 @@
+Unsplash is a photography sharing application. It has millions of users who post photos related to different topics every day. It is estimated to have more than two million photos. With the need to bring their services closer to developers, Unsplash released their developer API. With over six billion requests per month, it provides an infrastructure for developers to build an experience for their users using services provided by Unsplash. In this article, we will walk through the steps of creating an Unsplash developer account and implement various functionalities of the API on a Graphql API.
+
+#### prerequisites.
```

Comment:
remove the full stop. This applies to all the titles.

Reply:
fixed.